### PR TITLE
SCAN: Document steps to add new study arm

### PR DESCRIPTION
**Pull request by kairstenfay**
I know this work is still being scoped out, but I wanted to begin the documentation process of tracking all the steps required to add a new SCAN study arm. Feel free to comment / push to this PR while it's still a draft.

### CODE REVIEW

File: SCAN.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+## Adding a new study arm
+
+1. First, answer the questions listed in the [barcodes documentation] pertaining to creating new collections.
+2. Create a new identifier set.
+3. Create a new barcode label layout for the [labelmaker].
+4. Enable REDCap DETs for the new project once testing is complete and it's ready for production.
+5. Update the REDCap DET ETL with the new study's project ID and purview (see [example]).
+6. Add a new cronjob for the new REDCap DET ETL.
+7. ((placeholder for SCAN RoR workflow changes))
+8. Update the [export-record-barcodes] script in the [scan-switchboard] repo with the new language's project ID and purview. ((?))
```
Comment (kairstenfay) [inline SCAN.md:15]: Is the Switchboard for unboxing mail kits only, or would the lab like to look up any SCAN barcode?

**Comment by kairstenfay** [comment]
[New study arm for SCAN kiosks](https://trello.com/c/78XtEaMY/665-new-study-arm-for-scan-kiosks)

File: SCAN.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+## Adding a new study arm
+
+1. First, answer the questions listed in the [barcodes documentation] pertaining to creating new collections.
+2. Create a new identifier set.
+3. Create a new barcode label layout for the [labelmaker].
+4. Enable REDCap DETs for the new project once testing is complete and it's ready for production.
+5. Update the REDCap DET ETL with the new study's project ID and purview (see [example]).
+6. Add a new cronjob for the new REDCap DET ETL.
+7. Decide if we need to manually generate DETs (for e.g. if the new study arm uses the REDCap mobile app or API for enrollments)
+   > Note: even if the mobile app or API are being used for enrollments in this new study, as long as a field like `back_end_scan_date` is updated manually during QC, then we'll receive DETs as usual for this project.
```
Comment (tsibley) [inline SCAN.md:15]: > we'll receive DETs as usual for this project.

Not _quite_ DETs as usual, but we'll get at least one DET for the instrument containing the manually-updated field. If we have other required instruments, that DET must come _after_ those instruments are complete. This works fairly well if the manually-updated field is in a later instrument, like kit unboxing.

File: SCAN.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+2. Create a new identifier set.
+3. Create a new barcode label layout for the [labelmaker].
+4. Enable REDCap DETs for the new project once testing is complete and it's ready for production.
+5. Update the REDCap DET ETL with the new study's project ID and purview (see [example]).
+6. Update the FHIR ETL with the new study's collection identifier set.
+7. Add a new cronjob for the new REDCap DET ETL.
+8. Decide if we need to manually generate DETs (for e.g. if the new study arm uses the REDCap mobile app or API for enrollments)
+   > Note: even if the mobile app or API are being used for enrollments in this new study, as long as a field like `back_end_scan_date` is updated manually during QC, then we'll receive DETs as usual for this project.
+9. Update the manifest ETL expected collection identifier sets
+10. ((placeholder for SCAN RoR workflow changes))
```
Comment (kairstenfay) [inline SCAN.md:18]: + modify SCAN RoR View
+ update backoffice export-redcap-scan script

File: SCAN.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
 
+- [Adding a new study arm](#adding-a-new-study-arm)
+- [Adding a new language](#adding-a-new-language)
+
+## Adding a new study arm
+
+1. First, answer the questions listed in the [barcodes documentation] pertaining to creating new collections.
+2. Create a new identifier set.
+3. Create a new barcode label layout for the [labelmaker].
+4. Enable REDCap DETs for the new project once testing is complete and it's ready for production.
```
Comment (jccraft-uw) [inline SCAN.md:11]: I couldn't figure out where to do this in RedCAP. Let's add this to the step:
Within the REDCap project, go to Project Setup, find and click the Additional Customizations button, scroll to the bottom of the modal, and find the Data Entry Trigger section.

Comment (kairstenfay) [reply SCAN.md:11]: That make sense. I almost forget where that button is every time. This task appears in a couple of places, so this makes me inclined to create a REDCap documentation page where we describe tasks like this. I'll push a fix.

Comment (kairstenfay) [reply SCAN.md:11]: I added your steps to a [new wiki page](https://github.com/seattleflu/documentation/commit/e65c5a8d09afeefc9cd0e11ba808e7d22039ad2a).

File: SCAN.md (hunk 2 above)
Comment (kairstenfay) [reply SCAN.md]: I addressed this in the latest commit.

**Pull request merged**